### PR TITLE
Fixed abstract model declaration

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-3970

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @peter-fogg @bderusha

File: course_discovery/apps/course_metadata/migrations/0001_initial.py [hunk 1]
```diff
@@ -3,44 +3,29 @@
 
 from django.db import migrations, models
 import django_extensions.db.fields
-from django.conf import settings
-import sortedm2m.fields
 import django.db.models.deletion
+import sortedm2m.fields
+from django.conf import settings
```
Comment (peter-fogg) [inline course_discovery/apps/course_metadata/migrations/0001_initial.py:8]: These should be sorted.

Comment (peter-fogg) [inline course_discovery/apps/course_metadata/migrations/0001_initial.py:8]: Oh nevermind, I missed that this was a migration file. Carry on.

**Comment by peter-fogg** [comment]
:+1:

**Comment by bderusha** [comment]
:+1:

**Pull request merged**